### PR TITLE
ARMADA-461 handle exceptions properly at cli

### PR DESCRIPTION
**Pull request by fschuch**
It aims to provide a more user-friendly message.

#### What
The code should catch all the exceptions and handle them with good messages to users.

#### Why
It provides a better user experience

`Task`: https://app.clickup.com/t/18022949/ARMADA-461

---

#### Peer Review
Please follow the upstream omnivector documentation concerning
[peer-review guidelines](https://github.com/omnivector-solutions/Documentation/blob/main/Contributing/pr_review_standards.md#peer-review).

### CODE REVIEW

**Comment by jamesbeedy** [comment]
Very nice!

File: jobbergate-cli/jobbergate_cli/config.py [hunk 1]
```diff
@@ -50,7 +55,7 @@ class Settings(BaseSettings):
 
     IDENTITY_CLAIMS_KEY: str = "https://omnivector.solutions"
 
-    @root_validator
+    @root_validator(skip_on_failure=True)
```
Comment (matheushent) [inline jobbergate-cli/jobbergate_cli/config.py:58]: what's the purpose on skip on failure?

File: jobbergate-cli/jobbergate_cli/config.py [hunk 2]
```diff
@@ -87,4 +92,22 @@ class Config:
...
+    Return a Setting object and handle ValidationError with a message to the user.
+    """
+    try:
+        return Settings(*args, **kwargs)
+    except ValidationError:
+        terminal_message(
+            conjoin(
+                "A configuration error was detected.",
+                "",
+                f"[yellow]Please contact [bold]{OV_CONTACT}[/bold] for support and trouble-shooting",
```
Comment (matheushent) [inline jobbergate-cli/jobbergate_cli/config.py:106]: isn't needed to close the `[yellow]` statement?

Comment (fschuch) [reply jobbergate-cli/jobbergate_cli/config.py]: Absolutely, I will commit the fix

**Review by matheushent — APPROVED** [review]

**Pull request merged**